src/main/java/org/cascadebot/cascadebot/utils/ParserUtils.java

```diff
@@ -0,0 +1,45 @@
+package org.cascadebot.cascadebot.utils;
+
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ParserUtils {
+
+    private static final Pattern TIME_PARSER = Pattern.compile("(?<hours>\\d+)h|(?<minutes>\\d+)m|(?<seconds>\\d+)s");
+
+    public static long parseTime(String input, boolean onlyAllowOnce) {
```

Comment:
This should also account for a number like `5`, a usual time stamp like `1:05`

Reply:
*Disclaimer:* I made this method
What would 5 represent?
Usual timestamp is a valid point

Reply:
5 seconds. although this might be something we should make a poll for in the discord, but it's eater second or minute

Reply:
5 seconds is reasonable in this case imo since it's the smallest representable unit

Reply:
Note: I will do this outside of this PR to make sure it's done efficiently